assets/js/breadcrumbs.js

```diff
@@ -0,0 +1,23 @@
+const controllerBreadcrumbs = (function (jQuery) {
```

Comment:
Nit:

Please adjust your editor to use 4 spaces as default for JS. And re-format this file please.